Makefile

```diff
@@ -57,7 +55,7 @@ install: install-binary install-docs
 	# TODO(runcom)
 	#install -m 644 completion/bash/skopeo ${BASHINSTALLDIR}/
 
-install-binary: skopeo
+install-binary: ./skopeo
```

Comment:
*shrug* it works either way, let’s not drag this PR out any longer. I would still be interested to know though :)